src/components/admin/form-components/SearchableSelector.jsx

```diff
@@ -104,7 +104,7 @@ export default class SearchableSelector extends React.Component {
 
     const noObjectsMessage = (
       <span style={{ color: 'rgba(0, 0, 0, 0.3)' }}>
-        `No {this.props.mode} are currently assigned to this article`
```

Comment:
@Helw150 I noticed that these backticks were showing up in the UI, I guess that was just you forgetting that it was html and not JS here right? Also I feel like the header looks a bit weird of the staff section no... it stands out in a bad way compared to the rest? I'm not sure if we should just try and replicate the Material UI label css for that?

Reply:
Oop yeah on the backticks. I need to throw in Snapshots ASAP because they are quick&easy and would have allowed us to catch unintended UI bugs like this when refactoring. Modularized code should still render the same snapshot, which I failed here!

Agree on the header - we should use [TextFieldLabel](https://github.com/mui-org/material-ui/blob/v0.17.1/src/TextField/TextFieldLabel.js) for consistency with the rest of our Material-UI usage.